### PR TITLE
Add more interlinks to blogposts

### PR DESCRIPTION
**Pull request by shadre**

### CODE REVIEW

File: _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md [hunk 1]
```diff
@@ -85,4 +85,4 @@ If you would like to see how [Knapsack Pro helps split tests across parallel job
 
 ## Summary
 
-If you would like to learn more about testing with [Knapsack Pro](https://knapsackpro.com?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast) you can check other articles on our [blog](/) like [testing with Cypress test runner](/2018/run-javascript-e2e-tests-faster-with-cypress-on-parallel-ci-nodes).
+If you would like to learn more about testing with [Knapsack Pro](https://knapsackpro.com?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast) you can check other articles on our [blog](/) like [testing with Cypress test runner](/2018/run-javascript-e2e-tests-faster-with-cypress-on-parallel-ci-nodes). And in case you are contemplating using Travis CI, then this [comparison of Travis CI with other solutions](https://knapsackpro.com/ci_comparisons?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast#travis-ci) can be helpful for you.
```
Comment (ArturT) [inline _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md:88]: cool link with anchor https://knapsackpro.com/ci_comparisons#travis-ci

File: _posts/2019-03-27-how-to-merge-codeclimate-reports-for-parallel-jobs-ci-nodes.md [hunk 2]
```diff
@@ -121,6 +121,6 @@ blocks:
 
 ## Summary and Queue Mode for dynamic test suite split
 
-CI builds can be much faster thanks to leveraging parallel jobs on Semaphore CI 2.0 and CI parallelisation on any CI provider (<a href="/">see more parallelisation examples for your CI providers</a>). You can check <a href="https://knapsackpro.com?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=how-to-merge-codeclimate-reports-for-parallel-jobs-ci-nodes">Knapsack Pro tool for CI parallelisation</a> and learn more about Queue Mode and what problems it solves in below video.
+CI builds can be much faster thanks to leveraging parallel jobs on Semaphore CI 2.0 and CI parallelisation on any CI provider (<a href="/continuous_integration/">see more parallelisation examples for your CI providers</a>). You can check <a href="https://knapsackpro.com?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=how-to-merge-codeclimate-reports-for-parallel-jobs-ci-nodes">Knapsack Pro tool for CI parallelisation</a> and learn more about Queue Mode and what problems it solves in below video.
```
Comment (ArturT) [inline _posts/2019-03-27-how-to-merge-codeclimate-reports-for-parallel-jobs-ci-nodes.md:124]: good catch with `<a href="/continuous_integration/">`

File: _posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md [hunk 3]
```diff
@@ -203,7 +203,7 @@ KnapsackPro::Hooks::Queue.after_subset_queue do |queue_id, subset_queue_id|
 end
 {% endhighlight %}
 
-This example is based on [FAQ](https://github.com/KnapsackPro/knapsack_pro-ruby#how-to-use-junit-formatter).
+This example is based on [Knapsack Pro FAQ](https://github.com/KnapsackPro/knapsack_pro-ruby#how-to-use-junit-formatter).
```
Comment (ArturT) [inline _posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md:206]: we could link here to this page https://knapsackpro.com/faq/question/how-to-use-junit-formatter

File: _posts/2019-09-16-how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs.md [hunk 4]
```diff
@@ -116,4 +116,6 @@ If you would like to better understand how Queue Mode works in Knapsack Pro and
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/hUEB1XDKEFY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+Also check out another article describing the [parallelisation setup for GitHub Actions for Rails with MySQL, Redis and Elasticsearch](/2019/github-actions-ci-config-for-ruby-on-rails-project-with-mysql-redis-elasticsearch-how-to-run-parallel-tests). By the way, if you are currently considering moving to GitHub Actions, definitely check out our [Comparison of GitHub Actions to other CI solutions](https://knapsackpro.com/ci_comparisons?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs#github-actions).
```
Comment (ArturT) [inline _posts/2019-09-16-how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs.md:119]: Cool 👍

File: _posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md (hunk 3 above)
Comment (shadre) [reply _posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md]: Good idea. Updated.

**Pull request merged**